packages/mobile/src/appContext.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AppContext = React.createContext();
+
+export class ContextProvider extends React.Component {
```

Comment:
Lets rename this to match its purpose: `ContextProvider` -> `RerenderContextProvider`